### ISSUE TITLE
linter-haml does not work.

### ISSUE THREAD

**Issue by ycrepeau** [AtomLinter/linter-haml#87]
I have PURPOSELY made BIG "errors" in my haml file to check 'linter-haml' plug in. I had no error reported, no warning and no indication that 'linter-haml' has failed.

In my Terminal.app window, I have run the haml-lint (the software installed with gem) and got this:

```ShellSession
yjc:rails_5 ycrepeau$ haml-lint app/views/
warning: parser/current is loading parser/ruby23, which recognizes
warning: 2.3.1-compliant syntax, but you are running 2.3.0.
warning: please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
app/views/home/index.html.haml:5 [W] LineLength: Line is too long. [454/80]
app/views/home/index.html.haml:6 [W] RuboCop: unexpected token tCOLON
(Using Ruby 2.0 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)
app/views/home/index.html.haml:10 [W] LineLength: Line is too long. [119/80]
[... tons of errors, warning etc. with my haml files ...]
```

In the plugin preference (linter-haml), I have set the exact PATH of my haml-lint executable.

Nothing seems to work.

**Comment by Arcanemagus** [comment]
When you have a HAML file open can you run `Editor: Log Cursor Scopes` and report back what you get?

**Comment by ycrepeau** [comment]
Scopes at Cursor

text.haml
entity.name.tag.class.haml

**Comment by Arcanemagus** [comment]
If you open Developer Tools and place a breakpoint [here](https://github.com/AtomLinter/linter-haml/blob/v1.2.0/lib/linter.coffee#L63) is it ever hit?

**Comment by ycrepeau** [comment]
No.

However, while looking for the way to set a breakpoint (I have been using Atom for less than 1 week now), I have discovered a "Console". In the Console, it is written:

"'linter-haml.globalHamlLintYmlFile' could not set the default. Attempted default: undefined; Schema: {"description":"Full path to a global Haml lint file, if no other is found","type":"string"}"

In the linter-haml/lib/main.coffee I have this:

globalHamlLintYmlFile:
      description: 'Full path to a global Haml lint file, if no other is found'
      type: 'string'

AND NO VALUE. 

Should I update this file to have something like:

globalHamlLintYmlFile:
      description: 'Full path to a global Haml lint file, if no other is found'
      type: 'string'
      default: '.haml-lint.yml'

?

**Comment by Arcanemagus** [comment]
That is a bug, but I don't _think_ it's affecting this. Was the `linter` package installed for you?

**Comment by ycrepeau** [comment]
Not installed "for" me but "by" me.

At this time, atom linter package version: 1.11.4
haml-lint: 0.17.1

which haml-lint returns (actual path of the executable):
/Users/<myusername>/.rvm/gems/ruby-2.2.2/bin/haml-lint

**Comment by Janther** [comment]
I have the same issue here: running the latest atom, atom-linter, haml-linter, haml-lint, and rubocop.
Ruby version 2.3.0

**Comment by hamholla** [comment]
Same issue here... Coworker is also getting it.

**Comment by Arcanemagus** [comment]
I'm going to mark this as closed, v1.0.0 was released on 2016-08-06, which apparently updated this linter provider for the current Linter v1 API. If you are still experiencing problems please file a new issue.
